docs-ref-conceptual/use-azure-cli-successfully-quoting.md

```diff
@@ -128,7 +131,71 @@ PowerShell don't.
   az ad app create --display-name myName --native-app --required-resource-accesses @manifest.json
   ```
 
-If you are working in the PowerShell scripting language, see [Considerations for running the Azure CLI in a PowerShell scripting language - JSON strings](./use-azure-cli-successfully-powershell.md#pass-parameters-containing-json).
+Here are the accepted JSON format patterns for Bash, PowerShell and Cmd:
+
+### [Bash](#tab/bash2)
+
+Use Bash's `clear` command to remove console output between tests.
+
+```azurecli
+# Correct in Bash
+az '{"key":"value"}' --debug
+>> Command arguments: ['{"key":"value"}', '--debug']
+
+az "{\"key\":\"value\"}" --debug
+>> Command arguments: ['{"key":"value"}', '--debug']
+```
+
+These next two examples are **incorrect** as quotes and spaces are interpreted by Bash.
+
+|Incorrect format|Problem|Console output|
+|-|-|-|
+|az {"key":"value"} --debug |Missing single quotes or escape characters| Command arguments: ['{key:value}', '--debug']
+|az {"key": "value"} --debug |Missing single quotes or escape characters, and contains extra space | Command arguments: ['{key:', 'value}', '--debug']
+
+### [Powershell](#tab/powershell2)
+
+Use PowerShell's `cls` or `clear` command to remove console output between tests.
+
+```azurecli
+# Correct in PowerShell
+az '{"key": "value"}' --debug
+>> Command arguments: ['{key: value}', '--debug']
+
+az "{`"key`": `"value`"}" --debug
+>> Command arguments: ['{key: value}', '--debug']
+
+az "{""key"": ""value""}" --debug
+>> Command arguments: ['{key: value}', '--debug']
+```
+
+These next three examples are all **incorrect** in PowerShell.
+
+|Incorrect format|Problem |Console output|
+|-|-|-|
+|az "{\\"key\\":\\"value\\"}" --debug | Contains Bash escape characters | Command arguments: ['{\\', 'key\\:\\value\\}', '--debug']
+|az {"key":"value"} --debug | Missing single quotes | Unexpected token ':"value"' in expression or statement.
+|az {"key": "value"} --debug | Missing single quotes and contains an extra space | Error: Unexpected token ':' in expression or statement.
+
+### [Cmd](#tab/cmd2)
+
+Use the Cmd's `cls` command to remove console output between tests.
+
+```azurecli
+# Correct in Windows Command 
+az "{"key":"value"}" --debug
+>> Command arguments: ['{key:value}', '--debug']
+```
+
+These next three examples are all **incorrect** in Windows Command. 
+
+|Incorrect format|Problem |Console output|
+|-|-|-|
+|az "{"key":"value"}" --debug | Missing escape characters | Command arguments: ['{key:value}', '--debug']
```

Comment:
This incorrect `"{"key":"value"}"` is the same as the above `# Correct in Windows Command` command. This makes this doc self-contradictory.